src/Components/TrainingPage.js

```diff
@@ -18,28 +18,59 @@ export function getTrainingMinimalPlayersNumber(training) {
 
 export const TrainingPage = props => {
   const { navigation, route } = props;
-  const training = route.params.training;
+  const { training, program } = route.params;
 
   const onDrillPress = drill => navigation.navigate('DrillPageMinimal', { drill, training });
+  const goToFirstDrill = () => navigation.navigate('DrillPageMinimal', { drill: training.drills[0], training });
+
+  const programNavigation = () => {
+    const currentTrainingIndex = program.trainings.findIndex(programTraining => programTraining.id === training.id);
+    const isFirstTraining = currentTrainingIndex !== 0;
+    const isLastTraining = currentTrainingIndex !== program.trainings.length - 1;
+
+    const onNextPress = () =>
+      navigation.navigate('TrainingPage', { training: program.trainings[currentTrainingIndex + 1], program });
+    const onPrevPress = () =>
+      navigation.navigate('TrainingPage', { training: program.trainings[currentTrainingIndex - 1], program });
+    return (
+      <View style={styles.programNavigation}>
+        {isFirstTraining && (
+          <TouchableOpacity style={styles.btnPrevNext} onPress={onPrevPress}>
+            <Text style={styles.btnPrevNextContent}>{'<'}</Text>
+          </TouchableOpacity>
+        )}
+        <Text style={styles.title}>Training session {currentTrainingIndex + 1}</Text>
+        {isLastTraining && (
+          <TouchableOpacity style={styles.btnPrevNext} onPress={onNextPress}>
+            <Text style={styles.btnPrevNextContent}>{'>'}</Text>
```

Comment:
I think it should be nicer to use an icon, or a bigger font size for `<` and `>`

Reply:
Could it be possible to delete the words "minutes" and "players" or to keep it in the line with "12+" and "50+"?

Reply:
Having the training name to replace "training session 1" and the program name to replace "coed 2 long" would be better in my opinion.

Reply:
I changed it as you say, but I kep the training title in the header when browsing trainings (not from a program)

From a program :
<img width="481" alt="Screenshot 2020-05-03 at 13 52 37" src="https://user-images.githubusercontent.com/3810795/80913508-5ebe2180-8d45-11ea-93ba-35459b8bed8c.png">


From the trainings list :
<img width="477" alt="Screenshot 2020-05-03 at 13 50 02" src="https://user-images.githubusercontent.com/3810795/80913476-36362780-8d45-11ea-9e43-3403af4aee5d.png">


We loose the idea of progression within a program. You an navigate the trainings but you loose track of where you are within the program